faw/docker-bin/faw-db-console.py

```diff
@@ -1,14 +1,41 @@
 #! /usr/bin/env python3
 
 import code
+import multiprocessing.connection
 import os
 import pymongo
+import readline
+import rlcompleter
+import sys
 
 # Connect a pymongo instance with 'db' being the database corresponding to this
 # instance.
 _c = pymongo.MongoClient()
 db = _c[os.environ['DB']]
 
-print(f'Use the `db` variable to interact with the database')
-code.interact(local=dict(globals(), **locals()))
 
+def clear_database():
+    # Hardcoded port corresponds to (8123 + 2);
+    # If run in docker, the FAW server will listen on this port
+    conn = multiprocessing.connection.Client(('localhost', 8125))
```

Comment:
Some color on this approach: The cleanest ways to communicate between python programs are only possible when one is a subprocess of the other--this isn't the case in our current architecture. `multiprocessing.connection.Listener`/`multiprocessing.connection.Client`-based communication is the cleanest approach in general when programs are unrelated (in my opinion).

Another option would be to add a route to the existing web server for this functionality. This would need to be exposed on the same port as the rest of the http routes, so we couldn't hide it inside the docker container as we can here.

Reply:
In general, I like it. Should have some sort of parameter to opt-in to the non-reversible database reset.

Reply:
Agreed and done